packages/logger/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@empathyco/x-logger",
+  "version": "1.1.0",
+  "description": "Empathy simple logger",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./types/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc && tsc --project tsconfig.esm.json",
+    "build:watch": "tsc --watch",
+    "test": "jest"
+  },
+  "repository": {
```

Comment:
Is this property still necessary? If so, update the url pointing to this repository.